api/src/main/java/org/openmrs/module/eptsreports/reporting/utils/Indicators.java

```diff
@@ -0,0 +1,56 @@
+/**
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ *
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+package org.openmrs.module.eptsreports.reporting.utils;
+
+import org.openmrs.module.reporting.cohort.definition.CohortDefinition;
+import org.openmrs.module.reporting.evaluation.parameter.Mapped;
+import org.openmrs.module.reporting.evaluation.parameter.Parameter;
+import org.openmrs.module.reporting.indicator.CohortIndicator;
+import org.openmrs.module.reporting.indicator.aggregation.Aggregator;
+
+import java.util.Date;
+import java.util.Map;
+
+public class Indicators {
```

Comment:
Please move these indicators into the `library` folder

Reply:
[Done](https://github.com/esaude/openmrs-module-eptsreports/commit/17ed8b005b11bf93280c575532460c3da118831e).